packages/forms-web-app/src/controllers/selected-appeal/appeal-details/appeal-details-rows.js

```diff
@@ -1,52 +1,59 @@
 const { formatAddress } = require('@pins/common/src/lib/format-address');
 const {
-	formatApplicantDetails,
+	formatUserDetails,
 	formatHealthAndSafety,
 	formatProcedure,
-	formatLinkedAppeals,
+	formatRelatedAppeals,
 	formatYesOrNo,
-	formatContactDetails,
+	boolToYesNo,
 	formatAccessDetails,
 	formatDate
 } = require('@pins/common');
 const { APPEAL_USER_ROLES } = require('@pins/common/src/constants');
 
 /**
- * @typedef {import('appeals-service-api').Api.AppealCaseWithAppellant} AppealCaseWithAppellant
+ * @typedef {import('appeals-service-api').Api.AppealCaseDetailed} AppealCaseDetailed
  * @typedef {import("@pins/common/src/view-model-maps/rows/def").Rows} Rows
  */
 
 /**
- * @param {AppealCaseWithAppellant } caseData
+ * @param {AppealCaseDetailed } caseData
  * @param {string} userType
  * @returns {Rows}
  */
 
 exports.detailsRows = (caseData, userType) => {
 	const isAppellantOrAgent = userType === (APPEAL_USER_ROLES.APPELLANT || APPEAL_USER_ROLES.AGENT);
```

Comment:
Not a change from this PR, but this isn't right is it? 

Was the intention 

`const isAppellantOrAgent = userType === APPEAL_USER_ROLES.APPELLANT || userType === APPEAL_USER_ROLES.AGENT;`